src/Workspace/index.jsx

```diff
@@ -350,6 +378,23 @@ class Workspace extends React.Component {
     }
   }
 
+  checkWorkspacePayModel = async () => {
+    if (this.state.payModelInterval) {
+      clearInterval(this.state.payModelInterval);
+    }
+    try {
+      const payModelInterval = setInterval(async () => {
+        const data = await this.getWorkspacePayModels();
+        this.setState({
+          payModel: data,
+        });
+      }, 30000);
+      this.setState({ payModelInterval });
```

Comment:
`checkWorkspacePayModel` doesn't really need to be a periodical job for now

but it will be needed once we really stores and fetches the value of cost from DB